tests/Util/FormBuilderIteratorTest.php

```diff
@@ -51,6 +50,22 @@ public function testHasChildren(): void
     {
         $this->builder->add('name', TextType::class);
         $iterator = new FormBuilderIterator($this->builder);
-        static::assertTrue($iterator->hasChildren());
+        static::assertFalse($iterator->hasChildren());
```

Comment:
I've changed from using a stub of `FormFactoryInterface` to using an instance of `FormFactory`. This resulted in this test to fail. After checking what `hasChildren` does, I don't think this test was ever correct and only resulted in the expected result because it was a stub.
`hasChildren` checks if the children of current have children. This is not the case if we have one single `TextType`.

Reply:
Then can you add a hasChildren test when we have the true response too ?